### ISSUE TITLE
Where is the robin hood api documentation? (unofficial?)

### ISSUE THREAD

**Issue by rascoro1** [jtgagne/rtrade#1]
Are you using the following link?
https://github.com/sanko/Robinhood
I am looking to contribute and want to make sure I am using the same documentation you are using?

**Comment by jtgagne** [comment]
Yes, that was where I had started from. However, the documentation for gaining authentication is out of date since robinhood has switched to OAuth2. The new URLs for obtaining a token are used in RobinHood.cs. Also, I have not found a creative way of getting around TFA. You will need to disable it on your account to get a token using the code. Once you have a valid token on your local machine, you can re-enable TFA on your account and the token on your machine will still be valid for a day.
